build-support/migration-support/modernize_sources.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/env python3
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+"""A script to modernize the `sources` and `bundles` fields of BUILD files to use the new...
```

Comment:
```suggestion
"""A script to update `sources` and `bundles` fields of BUILD targets to use the new syntax
```

Also might `fix_deprecated_globs_in_targets.py` be a better fit for the script name?

Reply:
Oh oops. That was a TODO I missed. Thanks for flagging this!